src/client/renderer/r_lightmap.c

```diff
@@ -113,35 +114,40 @@ static void R_AllocStainmap_(const uint32_t width, const uint32_t height, r_stai
 	R_BindFramebuffer(FRAMEBUFFER_DEFAULT);
 }
 
-#define R_AllocLightmap(w, h) R_AllocLightmap_(IT_LIGHTMAP, w, h)
+#define R_AllocLightmap(w, h) R_AllocLightmap_(IT_LIGHTMAP, w, h, MAX_LIGHTMAP_PAGES)
```

Comment:
Is it pages, or layers? Or are those different things? Just for sake of consistency..

Reply:
PAGES = LAYERS, may rename it later

Reply:
Let's use LAYERS everywhere for consistency.